pages/index.js

```diff
@@ -191,12 +191,23 @@ function Index(props) {
 }
 
 Index.getInitialProps = async () => {
-  const res = await fetch(`${baseUrl}/twitch?reqType=isLive`).then(async (response) => {
-    const reply = await response.json();
-    return reply.response;
+  let isStreamLive;
+  let schedule;
+
+  const client = require('contentful').createClient({
+    space: process.env.CONTENTFUL_SPACE_ID,
+    accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+  });
+
+  await client.getEntries({
+    content_type: 'streamInfo',
+  }).then((res) => {
+    const { isLive, scheduleGraphic } = [...res.items][0].fields;
```

Comment:
What does the `[0]` here make reference to?

Reply:
/ is it possible we'd have more than one value in this response, and if so, is that something that'd confuse this logic?

Reply:
The inital getEntries call with return anything from contentful that has the content type "StreamInfo". Going to share with Link & anyone else that may be accessing this that to update the site he'll just need to edit the existing value that's in there and hit save. As long as we don't add in a new one, we can be sure that the first result is what we're looking for.